content/blog/2021/containerqueries.md

```diff
@@ -17,7 +17,6 @@ summary: |
   and proposing for years, has finally made its debut in a browser. Well, sort
   of. Here we'll explain what container queries are, how they work, and what
   other features they might come with once fully supported in browsers.
-home: true
```

Comment:
I removed all the `home` settings for now, but let me know if you want any of the 4 featured ones removed, or another post pinned.